Phase1/src/test/java/InvertedIndexTest.java

```diff
@@ -0,0 +1,59 @@
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+import java.util.TreeSet;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class InvertedIndexTest {
+    private static InvertedIndex invertedIndex;
+
+    @BeforeAll
+    public static void init() {
+        invertedIndex = new InvertedIndex();
+        invertedIndex.tokenizeFiles(new FileReader().readingFiles("src/test/testDocs"));
```

Comment:
Its better to mock the FileReader object to only test the Inverted Index class, not the others.